Apps/ValidationTests/Scripts/config.json

```diff
@@ -1,383 +1,389 @@
-            "referenceImage": "gltfMeshPrimitiveMode0.png",
-            "errorRatio": 8.0,
-            "excludedEngines": ["webgpu"]
-        },
-        {
-            "title": "GLTF Mesh Primitive Mode (1)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Mesh_PrimitiveMode, __page__, 1, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfMeshPrimitiveMode1.png",
-            "errorRatio": 8.0,
-            "excludedEngines": ["webgpu"]
-        },
-        {
-            "title": "GLTF Mesh Primitives",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Mesh_Primitives, __page__, 0, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfMeshPrimitives.png"
-        },
-        {
-            "title": "GLTF Mesh Primitives UV (0)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Mesh_PrimitivesUV, __page__, 0, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfMeshPrimitivesUV0.png",
-            "errorRatio": 5.0
-        },
-        {
-            "title": "GLTF Mesh Primitives UV (1)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Mesh_PrimitivesUV, __page__, 1, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfMeshPrimitivesUV1.png"
-        },
-        {
-            "title": "GLTF Node Attribute (0)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Node_Attribute, __page__, 0, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfNodeAttribute0.png"
-        },
-        {
-            "title": "GLTF Node NegativeScale (0)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Node_NegativeScale, __page__, 0, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfNodeNegativeScale0.png",
-            "errorRatio": 5.0
-        },
-        {
-            "title": "GLTF Node NegativeScale (1)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Node_NegativeScale, __page__, 1, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfNodeNegativeScale1.png"
-        },
-        {
-            "title": "GLTF Texture Sampler (0)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Texture_Sampler, __page__, 0, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfTextureSampler0.png",
-            "errorRatio": 8.0
-        },
-        {
-            "title": "GLTF Texture Sampler (1)",
-            "playgroundId": "#DS8AA7#27",
-            "replace": "__folder__, Texture_Sampler, __page__, 1, __disableSRGBBuffers__, 0",
-            "referenceImage": "gltfTextureSampler1.png"
-        },
-        {
-            "title": "GLTF Alien",
-            "playgroundId": "#XN37SR#5",
-            "referenceImage": "gltfAlien.png"
-        }
-    ]
+  "root": "https://cdn.babylonjs.com",
```

Comment:
Can you change the tab so it's easier to see the change?

Reply:
Reverted changes.